techniques/css/C45.html

```diff
@@ -0,0 +1,98 @@
+			<p>This satisfies the requirement of providing a <q>mode of operation where the keyboard focus indicator is visible</q>.</p>
+			<div class="note">
+				<p>This technique is only sufficient <em>if</em> it uses styles that provide a <em>visible</em> focus indicator.</p>
+			</div>
+			<div class="note">
+				<p>There may be situations where mouse/pointer users could also benefit from having a visible focus indicator, even though they did not set focus to an element using the keyboard. As a best practice, consider still providing an explicit <code class="language-css">:focus</code> style for these cases.</p>
+      </div>
+		</section>
+		<section id="examples">
+			<h2>Examples</h2>
+			<section class="example">
+				<h3>Using <abbr title="Cascading Style Sheets">CSS</abbr> <code class="language-css">:focus-visible</code> to provide keyboard focus indication</h3>
+				<p>In this example, we have a standard <code>&lt;button&gt;</code> control in the HTML.</p>
+        <p>By default, in all modern browsers, this button does not show any focus indication when activated using a mouse/pointer. Browsers show their default focus outline/indication when a user sets focus on the button using <kbd>Tab</kbd>.</p>
+        <p>In order to make this focus indication more prominent, we use the <code class="language-css">:focus-visible</code> pseudo-class selector to define a more intentional and pronounced focus style.</p>
+				<p>Working example: <a href="../../working-examples/css-focus-visible/">Using CSS <code class="language-css">:focus-visible</code> to provide keyboard focus indication</a>.</p>
+<pre><code>&lt;!DOCTYPE html&gt;
+&lt;html lang="en"&gt;
+&lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+  &lt;meta name="viewport" content="width=device-width,
+              initial-scale=1, shrink-to-fit=no"&gt;
+  &lt;title&gt;Using CSS :focus-visible to
+         provide keyboard focus indication&lt;/title&gt;
+  &lt;style&gt;
+    /* Specific :focus-visible styles */
+
+    :focus-visible {
+      outline: 5px solid red;
+    }
+    …
+  &lt;/style&gt;
+&lt;/head&gt;
+&lt;body&gt;
+  …
+  &lt;button&gt;Example button&lt;/button&gt;
+  …
+&lt;/body&gt;
+&lt;/html&gt;</code></pre>
+			</section>      
+		</section>
+		<section id="tests">
+			<h2>Tests</h2>
+			<section class="test-procedure">
+				<h3>Procedure</h3>
+				<p>For each user interface component that can receive keyboard focus:</p>
+				<ol>
+					<li>Set focus to the interface component using the keyboard (generally, navigating to the component using <kbd>Tab</kbd>/<kbd>Shift+Tab</kbd>)</li>
+          <li>Verify that once the component has received focus, there is a visible focus indication on or around the component</li>
```

Comment:
```suggestion
          <li>Verify that once the component has received focus, a focus indicator is visible</li>
```

Reply:
There's nothing normative about where the indicator is, so suggest keeping it short.

Reply:
Love it